### ISSUE TITLE
Requried App Runtime 1.5 Not availbale anymore

### ISSUE THREAD

**Issue by Thane5** [sotanakamura/winui-python#12]
Microsoft no longer offers Windows App Runtime 1.5 for download. Only versions beyond 1.6 are available. Therefore i can not run the demo projects.
![image](https://github.com/user-attachments/assets/e1316bce-a5b8-4274-b825-47f4b90a5f89)

**Comment by sotanakamura** [comment]
You can download Windows App SDK 1.5 in the following page.
https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads-archive

**Comment by sotanakamura** [comment]
I think 1.6 is compatible with 1.5. So you can use 1.6 instead of 1.5.

**Comment by Thane5** [comment]
i tried both 1.6 and 1.5 (1.5.0 and 1.5.7) None of them worked and the error popup came up everytime.

**Comment by sotanakamura** [comment]
It works in my environment. Did you install Python from https://www.python.org/?

**Comment by Thane5** [comment]
Yes. Initially i ran everything form a virtual environment in PyCharm, but now i also tested it with my system wide python installation and launched it via the console. Still, same error.

**Comment by sotanakamura** [comment]
Please run the following command and send the output

```
where python
```

and ensure that the runtime is installed by running the following command

```
powershell Get-AppxPackage -Name "*WindowsAppRuntime*"
```

**Comment by Thane5** [comment]
`where python` returned nothing

here are the app runtimes the other command found:
![image](https://github.com/user-attachments/assets/5010a557-c6fd-4d97-a92e-4fecd0ca9c34)

**Comment by sotanakamura** [comment]
Please use command prompt

**Comment by Thane5** [comment]
Ah sorry, here is what CMD found:

C:\Users\myname\AppData\Local\Programs\Python\Python312\python.exe
C:\Users\myname\AppData\Local\Microsoft\WindowsApps\python.exe

**Comment by sotanakamura** [comment]
Please run the following command in the cloned repo

```
python winui-xaml.py
```

**Comment by Thane5** [comment]
that makes the error popup appear,

**Comment by sotanakamura** [comment]
After reinstalling Windows app runtime, I have had the same problem. I will explore it a little bit.

**Comment by sotanakamura** [comment]
The same issue is reported.
https://github.com/microsoft/WindowsAppSDK/issues/4268

**Comment by sotanakamura** [comment]
It works on Windows 22H2.

Edition	Windows 11 Enterprise
Version	22H2
OS build	22621.4391

**Comment by sotanakamura** [comment]
@Thane5 Do you still have this problem? When I installed the runtime v1.6.5, I didn't have the problem. Please try to install the runtime v1.6.5 and let me know the result. If you sill have this problem, please share the version of Windows.

**Comment by luojunyuan** [comment]
Is there anyway to build with the dependency like msbuild property `<WindowsAppSDKSelfContained>`

**Comment by sotanakamura** [comment]
@luojunyuan You can get DLLs by downloading [the redistributable msix package](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads), renameing it to a zip file, and extract it. I think you need to put the DLLs in the proper directory, not in the directory where your script is stored.

**Comment by Thane5** [comment]
Did a quick test on another PC of mine. That example project finally opened for the first time!

**Comment by jasonm23** [comment]
I found using winget to be simpler

```
winget install --id=Microsoft.WindowsAppRuntime.1.6  -e
```
